src/checkers/inference/solver/backend/encoder/ConstraintEncoderCoordinator.java

```diff
@@ -12,7 +12,7 @@
 import checkers.inference.solver.backend.encoder.preference.PreferenceConstraintEncoder;
 
 /**
- * Created by mier on 08/11/17.
+ * A coordinator class that has the logic how each encoder encodes its supported constraint
```

Comment:
This seems dispatch constraints to the correct encoders based on the slot composition of the given constraint. So it is not "has the logic of how each encoder encodes constraint", but is "dispatch to the correct encoder to encode constraints".